lib/web_ui/lib/src/engine/semantics/semantics.dart

```diff
@@ -337,6 +338,9 @@ class SemanticsNodeUpdate {
 
   /// See [ui.SemanticsUpdateBuilder.updateNode].
   final int headingLevel;
+
+  /// See [ui.SemanticsUpdateBuilder.updateNode].
+  final String? linkUri;
```

Comment:
`linkUri` is non-nullable in the `updateNode` API. Does it need to be nullable here?

Reply:
This mirrors the code for the nullable `tooltip` field. I think it is non-nullable in the `updateNode` API to make it easier for the C++ side to handle.